### PR TITLE
Relax validation and add the endpoint_id

### PR DESCRIPTION
**Pull request by dantleech**
The existing validation rules do not correlate with the required fields in Onestock. The API should be agnostic of business logic.

### CODE REVIEW

File: src/Inviqa/OneStock/Order/OrderSanitizer.php [hunk 1]
```diff
@@ -9,30 +9,31 @@ class OrderSanitizer
...
         'title' => '',
+        'shipping_amount' => '',
+        'shipping_address_line_1' => '',
+        'shipping_city' => '',
+        'shipping_postcode' => '',
+        'shipping_country_code' => '',
+        'billing_address_line_1' => '',
+        'billing_city' => '',
+        'billing_postcode' => '',
+        'billing_country_code' => '',
```
Comment (dantleech) [inline src/Inviqa/OneStock/Order/OrderSanitizer.php]: onestock accepts an array of lines for the address -- so some of these, if not all, can be empty.

finally, Onestock reports it's own errors, so the more we leave to one stock, the more truth there is.

File: composer.json [hunk 2]
```diff
@@ -18,6 +18,13 @@
             "": ["src","features/bootstrap"]
         }
     },
+    "scripts": {
+        "integrate": [
+            "./bin/phpspec run --ansi",
```
Comment (elvetemedve) [inline composer.json]: `./` prefix can be omitted.

File: composer.json [hunk 3]
```diff
@@ -18,6 +18,13 @@
...
+            "./bin/behat --colors",
```
Comment (elvetemedve) [inline composer.json]: `./` prefix can be omitted.

File: bin/phpstan [hunk 4]
```diff
@@ -0,0 +1 @@
+../vendor/phpstan/phpstan/phpstan
```
Comment (dantleech) [inline bin/phpstan]: we should at least ignore the `bin` dir, and I'm not sure why these symlinks were not auto-generated by composer. I would simply leave them in `vendor/bin`

**Pull request merged**